### PR TITLE
Release notes for Percona Server 5.5.58-38.10 (5.5)

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.5.58-38.10.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+New Features
+============
+
+ |Percona Server| packages are now available for *Ubuntu 17.10 (Artful)*.
+
+Bugs Fixed
+==========
+
+ When shutting down the :ref:`threadpool` group, mutex in error branches were
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.58-38.10.rst]: If an I/O syscall returned an error during the server shutdown with threadpool enabled, a mutex could be left locked. Bug fixed, credit to Daniel Black.

File: doc/source/release-notes/Percona-Server-5.5.58-38.10.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ not unlocked. Bug fixed :bug:`1702330`.
+
+ ``MEMORY`` storage engine incorrectly allowed ``BLOB`` columns before indexed
+ columns. Bug fixed :bug:`1731483` (*Oleg Smirnov*).
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.58-38.10.rst]: We did not merge Oleg's patch (we merged and reverted it), yet his patch was what forced fixing the actual bug. Not sure how to describe this briefly.

File: doc/source/upstream-bug-fixes.rst [hunk 3]
```diff
@@ -139,31 +163,31 @@
...
 |:Launchpad bug: :bug:`1401791`                                                                               |
-|:Upstream state: Verified (checked on 2017-03-20)                                                            |
+|:Upstream state: Verified (checked on 2017-12-06)                                                            |
 |:Fix Released: :rn:`5.5.41-37.0`                                                                             |
 |:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`74440` - mysql_install_db not handling mysqld startup failure                      |
 |:Launchpad bug: :bug:`1382782`                                                                               |
-|:Upstream state: Won't Fix                                                                                   |
+|:Upstream state: Won't fix                                                                                   |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: Intentional?

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/installation/apt_repo.rst [hunk 4]
```diff
@@ -12,13 +12,15 @@ Supported Releases:
...
  * 8.0 (jessie)
+ * 9.0 (stretch)
 
 * Ubuntu:
 
- * 12.04LTS (precise)
  * 14.04LTS (trusty)
  * 16.04LTS (xenial)
  * 16.10 (yakkety)
+ * 16.04 (zesty)
```
Comment (laurynas-biveinis) [inline doc/source/installation/apt_repo.rst]: 17.04

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/installation/apt_repo.rst (hunk 4 above)
Comment (EvgeniyPatlan) [reply doc/source/installation/apt_repo.rst]: * 16.10 (yakkety) - shouldn't we remove it?

**Pull request merged**